docs/general/contributor-guide.md

```diff
@@ -0,0 +1,93 @@
+---
+title: Contributor Guide
+sidebar_position: 2
+---
+
+## Submit an issue
+
+1. Before submitting your issues, please go through a comprehensive search to make sure the problem cannot be solved just by searching.
+1. Check the Issue List to make sure the problem is not repeated.
+1. Create a new issue and choose the type of issue.
+1. Define the issue with a clear and descriptive title.
+1. Fill in necessary information according to the template.
+1. Choose a label after the issue is created.
+1. Please pay attention to your issue, you may need to provide more information during discussion.
+
+## Developer Flow
+
+### Fork repo
+
+Fork the BeyondStorage repo to your repo to work, then setting proper upstream.
+
+Take [go-storage](https://github.com/beyondstorage/go-storage) as an example:
+
+```shell
+# Clone the forked repo to local, replace the username with your github login.
+git clone git@github.com:<username>/go-storage.git
+# Setup beyondstorage/go-storage as a new remote called upstream.
+git remote add upstream git@github.com:beyondstorage/go-storage.git
+```
+
+### Choose an issue
+
+1. Please choose your target issue. If it is a new issue discovered or a new function enhancement to offer, please create an issue and set the right label for it.
+1. After choosing the relevant issue, please reply with a deadline to indicate that you are working on it.
+1. Depends on your working on projects, please ping different groups to give you feedback about your PR or issue in time.
+
+For go-storage related projects (including `go-storage` and `go-storage-*`):
+
+- `@go-storage-maintainer`
+- `@go-storage-committer`
+- `@go-storage-reviewer`
+
+For `BeyondTP`:
+
+- `@beyond-tp-maintainer`
+- `@beyond-tp-committer`
+- `@beyond-tp-reviewer`
+
+For other projects, there are no dedicated teams responsible for, our community members will try our best.
+
+### Create your branch
+
+Switch to your forked master branch, pull codes from upstream, then create a new branch.
+
+```shell
+git checkout master
+git pull upstream master
+git checkout -b IssueNo
+```
+
+### Coding
+
+1. Please obey the Code of Conduct during the process of development and finish the check before submitting the pull request.
+1. Then push codes to your fork repo.
+
+```shell
+git add modified-file-names
+git commit -m 'commit message'
+git push origin <IssueNumber>
+```
+
+### Submit PR
+
+1. Send a pull request to the master branch.
+1. The mentor will do a code review before discussing some details (including the design, the implementation, and the performance) with you.
+1. Also make sure that the pull request title has a semantic prefix like `fix:` or `feat:` or any other [conventional commit types](https://github.com/commitizen/conventional-commit-types/blob/master/index.json).
```

Comment:
How about swapping the order of 2 and 3? Does the PR need to add `fixes #IssueNo` or `closes #IssueNo` to close the issue?